docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,172 @@
+
+    The exact settings you need depend on what SASL mechanism your Kafka cluster is using and how your SSL certificates
+    are signed. For full details, see the `Security section of the Kafka documentation <http://kafka.apache.org/documentation.html#security>`__.
+
+#.  Start KSQL server with your configuration file specified.
+
+    .. code:: bash
+
+        $ <path-to-confluent>/bin/ksql-server-start <path-to-confluent>/etc/ksql/ksql-server.properties
+
+    .. tip:: After KSQL is started, you can view your settings with the ``SHOW PROPERTIES;`` KSQL command.
+
+---------------------------------------------------
+Using KSQL with a secured Confluent Schema Registry
+---------------------------------------------------
+
+If you are using KSQL with HTTPS to Confluent Schema Registry, you must set the ``ksql.schema.registry.url`` to HTTPS
+(as shown in the example configuration) and set the ``KSQL_OPTS`` environment variable to define the credentials to use
+when communicating with the Confluent Schema Registry. This must be run before starting the KSQL server.
+
+.. code:: bash
+
+    # Define KSQL security credentials when communicating with the Confluent Schema Registry via HTTPS
+    $ export KSQL_OPTS="-Djavax.net.ssl.trustStore=/etc/kafka/secrets/kafka.client.truststore.jks
+                        -Djavax.net.ssl.trustStorePassword=confluent
+                        -Djavax.net.ssl.keyStore=/etc/kafka/secrets/kafka.client.keystore.jks
+                        -Djavax.net.ssl.keyStorePassword=confluent"
+
+-------------------------------------------------
+Using KSQL with a Kafka Cluster Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
+interactive or non-interactive.
+
+.. tip:: For more information about ACLs see :ref:`kafka_authorization` and for more information about interactive and
+         non-interactive queries, see :ref:`restrict-ksql-interactive`.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Interactive KSQL clusters require that the KSQL user has open access to create, read, write and delete topics and
+use any consumer group.
+
+The required ACLs are:
+
+- *DESCRIBE_CONFIGS* permission on the *CLUSTER*.
+- *CREATE* permission on the *CLUSTER*.
+- *DESCRIBE*, *READ*, *WRITE* and *DELETE* permissions on the *<any>* *TOPIC*.
+- *DESCRIBE* and *READ* permissions  on the *<any>* *GROUP*.
+
+It is still possible to restrict the KSQL user from accessing specific resources using *DENY* ACLs. For example, you can add a
+*DENY* ACL to stop KSQL queries from accessing a topic that contains sensitive data.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Non-Interactive KSQL clusters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Non-interactive KSQL clusters run with much more restrictive ACLs, though it currently requires a bit of effort to
```

Comment:
>Non-interactive KSQL clusters run with much more restrictive ACLs, though it currently requires a bit of effort to determine what ACLs are required
 
Doesn't read right to me, (and I wrote it! lol).

`Non-interactive KSQL clusters _can be_ run with more...`